src/vanilla/store.ts

```diff
@@ -199,13 +199,13 @@ const addPendingFunction = (pending: Pending, fn: () => void) => {
 }
 
 const flushPending = (pending: Pending) => {
-  let error: unknown | undefined
+  const ref: { e?: AnyError } = {}
```

Comment:
Let's avoid using extra object.
```suggestion
  let error: unknown | undefined
  let hasError = false
```